### PR TITLE
Fix the bug that models were not selected properly

### PR DESCRIPTION
**Pull request by d-hayashi**
## What?
- Fix method `is_loadable` to fix the behavior when either `contents`, `data_type`, or `data_type` is None
- Add tests for `BaseFileReader._select_model`

## Why?
Bug fix

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/pydtk/pull/103#" title="Author self-approved">d-hayashi</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fpydtk).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/pydtk/blob/master/OWNERS)~~ [d-hayashi]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by yadach — APPROVED** [review]
LGTM!!

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/pydtk/pull/103#" title="Author self-approved">d-hayashi</a>*, *<a href="https://github.com/dataware-tools/pydtk/pull/103#pullrequestreview-820982556" title="Approved">yadach</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fpydtk).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/pydtk/blob/master/OWNERS)~~ [d-hayashi,yadach]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
